README.markdown

```diff
@@ -64,7 +64,7 @@ Note: Now start vim with plain `vim`, not `vim .`
 > How can I open NERDTree automatically when vim starts up on opening a directory?
```

Comment:
For the benefit of new users, add some text to the question explaining what `exe 'cd '.argv()[0]` is doing in your snippet.

Reply:
Added a brief explanation of the `exe 'cd '.argv()[0]` command